### ISSUE TITLE
feat: improve cards navigation on Bevy3D

### ISSUE THREAD

**Issue by adolfousier** [adolfousier/agentverse#2]
feat: improve agents card navigation on Bevy3D.

Missing keyboard navigation on GPU's
